packages/search/app/styles/navi-search.less

```diff
@@ -3,157 +3,205 @@
  * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
  */
 
-@font-face {
-  font-family: 'denali-icon-font';
-  font-style: normal;
-  font-weight: normal;
-  src: url('denali-icon-font.eot?1576615905868');
-  src: url('denali-icon-font.eot?#iefix-1576615905868') format('embedded-opentype'),
-    url('denali-icon-font.svg?1576615905868') format('svg'), url('denali-icon-font.woff?1576615905868') format('woff'),
-    url('denali-icon-font.ttf?1576615905868') format('truetype');
-}
-
-.d-icon[class^='d-'],
-.d-icon[class*=' d-'] {
-  font-family: 'denali-icon-font' !important;
-  font-size: 24px;
-  font-style: normal;
-  font-variant: normal;
-  font-weight: normal;
-  line-height: 1;
-  speak: none;
-  text-decoration: inherit;
-  text-rendering: optimizeLegibility;
-  text-transform: none;
-  vertical-align: middle;
-  /* Better Font Rendering */
-  -webkit-font-smoothing: antialiased;
+.navi-search,
+.navi-search-results,
+.navi-search-results__item {
+  border-collapse: collapse;
+  color: @denali-grey-800;
+  font-family: Helvetica, 'Arial', sans-serif;
```

Comment:
Can we use one of the font families in `navi-core` or add this one as a variable there?